src/components/TodoForm/TodoForm.tsx

```diff
@@ -0,0 +1,129 @@
+import React, { useState } from 'react';
+import { User } from '../../types/User';
+
+const DEFAULT_INPUTS = {
+  title: '',
+  userId: 0,
+};
+
+const DEFAULT_INPUTS_ERRORS = {
+  title: false,
+  userId: false,
+};
+
+interface Props {
+  onAdd: (newTodoInfo: {
+    title: string,
+    userId: number,
+  }) => void;
```

Comment:
Should be the same type as for the `handleAddTodo`, so it's a good idea to create and use it. As I mentioned above